daemon/src/wallpaper_info.rs

```diff
@@ -12,6 +12,17 @@ pub struct WallpaperInfo {
     pub duration: Option<Duration>,
     #[serde(rename = "apply-shadow")]
     pub apply_shadow: Option<bool>,
+    #[serde(default)]
+    pub sorting: Sorting,
+}
+
+#[derive(Debug, Copy, Clone, Default, Eq, PartialEq,Deserialize)]
+#[serde(rename = "kebab-case")]
```

Comment:
Does it work without renaming? Also, do `natural` and `reverse` (in lowercase) work?

Reply:
Serde is magic to me. I freely admit to cargo-culting some of this. Happy to dig in and do some further testing. I do know that lowercase strings do not work, so I'd like to get that fixed up, too.